khronicle-core/src/commonTest/kotlin/LogTests.kt

```diff
@@ -217,6 +218,8 @@ class LogTests {
             metadata[StringKey] = "test"
             "Test message"
         }
+        // Double-checks that the `log` function containing the assert
```

Comment:
I'll update the comment. The `assertEquals` in this test is hidden inside of `logger.log`, which is called indirectly by the call to `Log.verbose`. I've had tests incorrectly pass in the past when using this pattern because the indirect call never actually happened, so the `assert` was never called (and could therefore never fail).

Reply:
I see, thanks for clarifying!